python-avocado.spec

```diff
@@ -148,6 +148,7 @@ popd
 %{__mkdir} -p %{buildroot}%{_mandir}/man1
 %{__install} -m 0644 man/avocado.1 %{buildroot}%{_mandir}/man1/avocado.1
 %{__install} -m 0644 man/avocado-rest-client.1 %{buildroot}%{_mandir}/man1/avocado-rest-client.1
+%{__install} -d -m 0755 %{buildroot}%{_sharedstatedir}/avocado/data
```

Comment:
I asked for clarification and maybe even suggestions on how to deal with this. I'm fine with merging this (even though there is a slight feeling saying it's not 100% clear).

As for proper solutions I'd either create a post-install script (or how is it called in SPEC file language) which updates the `/etc/avocado/avocado.conf` value based on the `%{_sharedstatedir}`, or I'd hardcode the `/var/lib` value as that is what we are using in config file.

All three would work for me so mainly this is me raising questions to avoid future issues.